SonarRunner.Shim/Resources.resx

```diff
@@ -148,7 +148,7 @@
     <value>Expected to be called with one argument: the full path to the analysis config file</value>
   </data>
   <data name="ERR_NoProjectInfoFilesFound" xml:space="preserve">
-    <value>No ProjectInfo.xml files were found. Check that the analysis targets are referenced by the MSBuild projects being built.</value>
```

Comment:
Since copying the targets files is now handled automatically, the removed error message is no longer actionable